src/styles/utils/functions.scss

```diff
@@ -0,0 +1,3 @@
+@function center-stopwatch-elements($size) {
+  @return ($stopwatch-size * 0.5) - ($size * 0.5);
```

Comment:
It looks like the variable `$stopwatch-size` is being used without being defined within this function or passed as an argument. Make sure that all variables used in a function are available in its scope, either by passing them as arguments or by importing them if they are global.